packages/jobs/package.json

```diff
@@ -25,20 +25,18 @@
         "@nangohq/records": "file:../records",
         "@nangohq/utils": "file:../utils",
         "@nangohq/webhooks": "file:../webhooks",
-        "@trpc/client": "^10.44.1",
-        "@trpc/server": "^10.44.1",
-        "@types/fs-extra": "^11.0.1",
+        "@trpc/client": "10.45.2",
+        "@trpc/server": "10.45.2",
         "axios": "^1.3.4",
         "dd-trace": "5.2.0",
         "dotenv": "^16.0.3",
-        "fs-extra": "^11.1.1",
+        "get-port": "7.1.0",
         "node-cron": "3.0.3",
-        "superjson": "^2.2.1",
```

Comment:
note: i am gonna get rid of trpc/superjson in the refactor of jobs. It is annoying to have 2 trpc services depend on each other since they have to import the client from each other. I am gonna use the same approach than in the orchestrator using express with typed endpoint definitions

Reply:
amazing news 🙌🏻